projects/kit/src/lib/masks/time/utils/parse-time.ts

```diff
@@ -0,0 +1,34 @@
+import {DEFAULT_TIME_SEGMENT_MAX_VALUES} from '../../../constants';
+import type {MaskitoTimeSegments} from '../../../types';
+import {parseTimeString} from '../../../utils/time';
+import type {MaskitoTimeParams} from '../maskito-time-params';
+
+export function maskitoParseTime(
+    maskedTime: string,
+    {
+        mode,
+        timeSegmentMaxValues = {},
+    }: Pick<MaskitoTimeParams, 'mode' | 'timeSegmentMaxValues'>,
+): number | null {
+    if (maskedTime === '') {
+        return null;
```

Comment:
Should we return `NaN` for empty string ? 🤔

Reply:
I removed `if` altogether. Now the empty string is parsed as `0`.